packages/lab/src/parent-child-layout/ParentChildLayout.tsx

```diff
@@ -74,53 +64,30 @@ export const ParentChildLayout = forwardRef<
 
   const isCollapsed = useIsViewportLargerThanBreakpoint(collapseAtBreakpoint);
 
-  const collapsedViewChildren = {
-    parent: (
-      <FlexItem
-        className={clsx(withBaseName("parent"), {
-          ["saltFlexItem-stacked"]: isCollapsed,
-        })}
-      >
-        {parent}
-      </FlexItem>
-    ),
-    child: (
-      <FlexItem
-        className={clsx(withBaseName("child"), {
-          ["saltFlexItem-stacked"]: isCollapsed,
-        })}
-      >
-        {child}
-      </FlexItem>
-    ),
-  };
-
   useEffect(() => {
     onCollapseChange?.(isCollapsed);
   }, [isCollapsed, onCollapseChange]);
 
   return (
-    <FlexLayout
-      ref={ref}
-      className={clsx(
-        withBaseName(),
-        {
-          [withBaseName(`no-animations`)]: disableAnimations,
-          [withBaseName(`reversed`)]: parentPosition === "right",
-        },
-        className
-      )}
-      gap={gap}
-      {...rest}
-    >
+    <div ref={ref} className={clsx(withBaseName(), className)} {...rest}>
       {isCollapsed ? (
-        collapsedViewChildren[collapsedViewElement]
```

Comment:
i think this approach makes sense if there were more than just a child vs parent element, but as there are only two possibilities i think a boolean is preferred ?